README.md

```diff
@@ -414,6 +414,19 @@ for other options).</p>
 at the end of the command can be used, but that should cover most possible use cases.</p>
     </td>
  </tr>
+ <tr>
+    <td><tt>extraunloadoptions</tt></td>
+    <td>No</td>
+    <td><tt>ESCAPE</tt> <tt>MANIFEST</tt> </td>
```

Comment:
Also, `spark-redshift` won't work unless `MANIFEST` is set, so I don't want to let users remove it.